### PR TITLE
test: Map widget automation

### PR DESCRIPTION
**Pull request by sarojsarab**
## Description
- Map widget automation

#### Type of change
- Added spec to verify the map widget with all the possible settings
- Updated the workflows to include map widget spec for hosted runs and not run for normal runs
#### How Has This Been Tested?
- [x] Cypress
>
>
#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#speedbreakers-) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans#areas-of-interest-)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [ ] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

**Comment by sarojsarab** [comment]
Sample [workflow run ](https://github.com/appsmithorg/appsmith/actions/runs/6464562501)

File: .github/workflows/ci-test-hosted.yml [hunk 1]
```diff
@@ -311,6 +311,13 @@ jobs:
           name: results-${{github.run_attempt}}
           path: ~/results
 
+      - name: Upload cypress snapshots
```
Comment (Aishwarya-U-R) [inline .github/workflows/ci-test-hosted.yml:314]: Was this not present til now?

**Review by Aishwarya-U-R — COMMENTED** [review]
Cypress changes for MapWidget looks good 👍

**Comment by Aishwarya-U-R** [comment]
> workflow run

Please run again & tag the Cypress dashboard link too for easier check.

File: .github/workflows/ci-test-hosted.yml (hunk 1 above)
Comment (sarojsarab) [reply .github/workflows/ci-test-hosted.yml:314]: nope

**Pull request merged**